src/commcare_cloud/commands/terraform/tests/test_compact_regexes.py

```diff
@@ -0,0 +1,92 @@
+import re
+
+from commcare_cloud.commands.terraform.constants import COMMCAREHQ_XML_POST_URLS_REGEX
+from commcare_cloud.commands.terraform.terraform import compact_waf_regexes_simply, compact_waf_regexes
+
+
+def test_compact_waf_regexes_simply__single_pattern():
+    assert compact_waf_regexes_simply([r'^/a/([\w\.:-]+)/api/v([\d\.]+)/form/$']) \
+           == [r'^/a/([\w\.:-]+)/api/v([\d\.]+)/form/$']
+
+
+def test_compact_waf_regexes_simply__two_patterns():
+    assert compact_waf_regexes_simply([
+        r'^/a/([\w\.:-]+)/api/v([\d\.]+)/form/$',
+        r'^/a/([\w\.:-]+)/api/v0\.6/case(?:/([\w\-,]+))?/?$'
+    ]) == [
+        r'^/a/([\w\.:-]+)/api/v([\d\.]+)/form/$|^/a/([\w\.:-]+)/api/v0\.6/case(?:/([\w\-,]+))?/?$'
+    ]
+
+
+def test_compact_waf_regexes_simply__match_examples():
+    _test_compact_function_against_examples(compact_waf_regexes_simply, COMMCAREHQ_XML_POST_URLS_REGEX)
+
+
+def test_compact_waf_regexes__single_pattern():
+    assert compact_waf_regexes([r'^/a/([\w\.:-]+)/api/v([\d\.]+)/form/$']) \
+           == [r'^/a/([\w\.:-]+)/(api/v([\d\.]+)/form/)$']
+
+
+def test_compact_waf_regexes_simply__two_patterns():
```

Comment:
I guess this test is the same as https://github.com/dimagi/commcare-cloud/pull/5525/files#diff-ddbd1b76aaed0c6cb3a6edb4f653f7d341bd3dbca2aa06091ad30fba48cd3dc9R12?

I am wondering why stickler did not complain about it. 🤔